tests/karate/src/test/java/be/features/decentralizedCom/heartbeat.feature

```diff
@@ -10,22 +10,25 @@ Feature: Send heartbeats to other servers
     * call read(serverFeature)
     * call read(mockClientFeature)
     * def mockServer = call createMockClient
-    * def lao = mockServer.createValidLao()
-    * def validRollCall = mockServer.createValidRollCall(lao)
+    * def mockFrontend = call createMockClient
+    * def lao = mockFrontend.createValidLao()
+    * def validRollCall = mockFrontend.createValidRollCall(lao)
 
     # This call executes all the steps to create a valid lao on the server before every scenario
     # (lao creation, subscribe, catchup)
-    * call read(createLaoScenario) { organizer: '#(mockServer)', lao: '#(lao)' }
+    * call read(createLaoScenario) { organizer: '#(mockFrontend)', lao: '#(lao)' }
 
-  # After lao creation, wait and do nothing (30 seconds for now) and check that a heartbeat message was received
+  # After lao creation, wait and do nothing (40 seconds for now) and check that more than one heartbeat message was received.
+  # (The initial one would be a response to publishing lao creation)
   Scenario: Server should send heartbeat messages automatically after a time interval
-    Given wait(30)
+    Given wait(40)
 
-    When def heartbeatMessages = mockServer.getMessagesByMethod('heartbeat')
+    When def heartbeatMessages = mockServer.getHeartbeats()
 
-    Then assert heartbeatMessages.length > 0
+    Then assert heartbeatMessages.length > 1
```

Comment:
Would be better to check for the actual number? Or at least if we cannot determine it precisely we could add an upper threshold (just to be sure heartbeats are not spammed more than expected)

Reply:
I was thinking to keep it more open in case it changes in the future, but right now it can absolutely just check that the number is 2.